replikator-api.go

```diff
@@ -142,9 +144,12 @@ func startApiServer() {
 
 func loggingMiddleware(next http.Handler) http.Handler {
 	return http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
-		log.Printf("[%s] %s %s", r.Method, r.RequestURI, r.RemoteAddr)
+		id := rand.Intn(99999-10000) + 10000
```

Comment:
Why the +10000?

Reply:
+10000 is added to make sure it's always 5 digits.

```	
        fmt.Println(rand.Intn(99999-10000) + 10000)
	fmt.Println(rand.Intn(99999))
```

Can output:
```
99516
925
```